Source/Urho3D/Network/Connection.cpp

```diff
@@ -1348,7 +1353,7 @@ void Connection::ProcessExistingNode(Node* node, NodeReplicationState& nodeState
         {
             Component* component = components[i];
             // Check if component is not to be replicated
-            if (!component->IsReplicated())
+            if (component->IsReplicated())

```

Comment:
And this is wrong too. Yes, I'm going to write comments for _all_ places with functional defects, regardless of similarity. Just to be sure they won't be lost.